test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -561,4 +567,97 @@ public void testIncludeAllColumnsAsMetadataFalse_mcSet() throws Exception {
     assertEquals("gsa1", adaptor.metadataColumns.getMetadataName("db_col1"));
     assertEquals("gsa2", adaptor.metadataColumns.getMetadataName("col2"));
   }
+
+  private void addDBTableForTest() throws SQLException {
+    executeUpdate("create table employees("
```

Comment:
Please match the other tables (see ResponseGeneratorTest) and just do something like
"create table data (id int, name varchar)". The extra column isn't adding value.